python/ray/serve/_private/common.py

```diff
@@ -354,6 +373,7 @@ class RunningReplicaInfo:
     actor_handle: ActorHandle
     max_concurrent_queries: int
     is_cross_language: bool = False
+    app_name: Optional[str] = None
```

Comment:
To confirm, is `app_name` optional solely for backwards compatibility with Ray 1.x? Under what conditions would `app_name` be `None`?